### DISCUSSION TITLE
Learning Cohorts

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#164] [Ideas]
Let's try to breakdown some ideas for what a 4-6 week long learning cohort could look like. 

Some things to consider:
- Should we cap the number of participants?
- Start and end date?
- Async, sync, choose your own adventure?
- Location?

**Comment by ClJarvis** [comment]
*  Cap might be good for the initial run just to help smooth out any kinks. Then expand the cap id there is interest. 
*  Async, sync, choose your own adventure * Sync would be good for people needing immediate feedback. Or for a more Hands on teaching event. But Async might be better due to the range of time zone and availability of VC members.

**Comment by adiati98** [comment]
I think we should know the background first.
Some of us have the same course and haven't started/finished it.  We wanted to gather a group to learn and tackle this together, but somehow it never happens.
The reasons: time differences, lost motivation, imposter syndrome, etc.

My idea is not to make this like a bootcamp. The goal is *everybody can have fun & move forward while learning together*. 
- Same topics/materials & pace so that no one would feel "left behind" and whoever understands something earlier can help the other. -  This would help in minimizing imposter syndrome and avoiding lost motivation.
- We can cap the number of participants. When it's not too big it would be more effective I guess (maybe something like 4 to 6)? We can make more than 1 group depend on the interest? Or add more seats when there's more interest if we want to have 1 group only.
- We can do async to accommodate time differences, but also open to sync. We can do a weekly or bi-weekly meetup to check in/recap the learning?
- Start and end date: TBA - We need to know first how many members are interested, which course/topic to learn, etc.
- Location: Can't think of it yet 😆

For self-taught (never been in a bootcamp), it would be nice to have the experience of:
- Accountable buddies
- Mentorship - Anyone who would like to volunteer in mentoring
- Pair programming - It would be less intimidating I believe to pair with people at the same level (but also maybe with attendance of a more experienced dev to guide?). Maybe like creating a mini project and do pair programming.

**Reply by BekahHW** [reply]
Anyone have thoughts about doing this with Just JavaScript?

**Reply by tkshill** [reply]
I think that's a good idea

**Reply by danieljanderson** [reply]
I like this idea.  We should do one on testing
